sechub-wrapper-owasp-zap/src/test/java/com/mercedesbenz/sechub/zapwrapper/config/ApiDefinitionFileProviderTest.java

```diff
@@ -14,127 +15,159 @@
+        List<File> result = providerToTest.fetchApiDefinitionFiles("test/path", sechubScanConfiguration);
 
         /* test */
-        assertEquals(path.toString(), "test/path/openapi3.json");
+        assertEquals(1, result.size());
     }
 
-    static Stream<Arguments> sourcesPartSizeTestNamedArguments() {
-        /* @formatter:off */
-        return Stream.of(
-        		Arguments.of(
-        				Named.of("Sources part empty",
-        				"{\"apiVersion\":\"1.0\",\"data\":{\"sources\":[]},\"webScan\":{\"uri\":\"https://localhost:8443\",\"api\":{\"type\":\"openApi\",\"use\":[\"open-api-file-reference\"]}}}")),
-        		Arguments.of(
-        				Named.of("Sources part more than one file",
-        				"{\"apiVersion\":\"1.0\",\"data\":{\"sources\":[{\"name\":\"open-api-file-reference\",\"fileSystem\":{\"files\":[\"openapi3.json\"]}},{\"name\":\"second-reference\",\"fileSystem\":{\"files\":[\"second-openapi-file.json\"]}}]},\"webScan\":{\"uri\":\"https://localhost:8443\",\"api\":{\"type\":\"openApi\",\"use\":[\"open-api-file-reference\",\"second-reference\"]}}}")),
-        		Arguments.of(
-        				Named.of("Binaries part used instead of sources",
-        				"{\"apiVersion\":\"1.0\",\"data\":{\"binaries\":[{\"name\":\"open-api-file-reference\"}]},\"webScan\":{\"uri\":\"https://localhost:8443\",\"api\":{\"type\":\"openApi\",\"use\":[\"open-api-file-reference\"]}}}")));
-        /* @formatter:on */
+    @ParameterizedTest
+    @MethodSource("multipleFilesTestNamedArguments")
+    void mutliple_files_result_in_correct_list_of_files(String sechubScanConfigJSON) {
+        /* prepare */
+        SecHubScanConfiguration sechubScanConfiguration = SecHubScanConfiguration.createFromJSON(sechubScanConfigJSON);
+
+        /* execute */
+        List<File> result = providerToTest.fetchApiDefinitionFiles("test/path", sechubScanConfiguration);
+
+        /* test */
+        assertEquals(2, result.size());
     }
 
-    static Stream<Arguments> filesystemPartSizeTestNamedArguments() {
+    static Stream<Arguments> multipleFilesTestNamedArguments() {
         /* @formatter:off */
         return Stream.of(
-        		Arguments.of(
-        				Named.of("Filesystem part empty",
-        				"{\"apiVersion\":\"1.0\",\"data\":{\"sources\":[{\"name\":\"open-api-file-reference\",\"fileSystem\":{}}]},\"webScan\":{\"uri\":\"https://localhost:8443\",\"api\":{\"type\":\"openApi\",\"use\":[\"open-api-file-reference\"]}}}")),
-        		Arguments.of(
-        				Named.of("Filesystem part more than one file",
-        				"{\"apiVersion\":\"1.0\",\"data\":{\"sources\":[{\"name\":\"open-api-file-reference\",\"fileSystem\":{\"files\":[\"openapi3.json\", \"second-file.json\"]}}]},\"webScan\":{\"uri\":\"https://localhost:8443\",\"api\":{\"type\":\"openApi\",\"use\":[\"open-api-file-reference\"]}}}")));
+              Arguments.of(
+                      Named.of("Sources part more than one file",
```

Comment:
Please, use variables before the `return` statement to store the sechub configs into and use text blocks.